interfaces/cython/cantera/cti2yaml.py

```diff
@@ -0,0 +1,1648 @@
+    """
+    pure_fluids = {
+        0: 'water',
+        1: 'nitrogen',
+        2: 'methane',
+        3: 'hydrogen',
+        4: 'oxygen',
+        5: 'HFC134a',
+        7: 'carbondioxide',
+        8: 'heptane'
+    }
+
+    def __init__(self, name='', elements='', species='', note='',
+                 substance_flag=0, initial_state=None, options=[]):
+
+        phase.__init__(self, name, 3, elements, species, note, 'none',
+                       initial_state, options)
+        self.thermo_model = 'pure-fluid'
+        self._subflag = substance_flag
+
+    def get_yaml(self, out):
+        super(liquid_vapor, self).get_yaml(out)
+        if self._subflag in self.pure_fluids:
+            out['pure-fluid-name'] = self.pure_fluids[self._subflag]
+        else:
+            raise InputError('liquid_vapor: unrecognized value "{}" for '
+                '"substance_flag"', self._subflag)
+
+
+class pureFluidParameters(object):
+    def __init__(self, species=None, a_coeff=[], b_coeff=0):
+        self._species = species
+        self._acoeff = a_coeff
+        self._bcoeff = b_coeff
+
+
+class crossFluidParameters(object):
+    def __init__(self, species=None, a_coeff=[], b_coeff=[]):
+        self._species1, self._species2 = species.split(' ')
+        self._acoeff = a_coeff
+        self._bcoeff = b_coeff
+
+
+class RedlichKwongMFTP(phase):
+    """A multi-component fluid model for non-ideal gas fluids.
+        """
+
+    def __init__(self, name='', elements='', species='', note='',
+                 reactions='none', kinetics='GasKinetics', initial_state=None,
+                 activity_coefficients=None, transport='None', options=[]):
+
+        phase.__init__(self,name, 3, elements, species, note, reactions,
+                       initial_state,options)
+        self.thermo_model = 'Redlich-Kwong'
+        self._kin = kinetics
+        self._tr = None if transport == 'None' else transport
+        self._activityCoefficients = activity_coefficients
+
+    def get_yaml(self, out):
+        super(RedlichKwongMFTP, self).get_yaml(out)
+        for section, names in self._sp:
+            if section != 'species':
+                _printerr("WARNING: Converting Redlich-Kwong species from"
+                    " different input files ({}) is not supported.".format(section))
+
+        spdict = {sp._name: sp for sp in _species}
+        for params in self._activityCoefficients:
+            if isinstance(params, pureFluidParameters):
+                sp = spdict[params._species]
+                sp.rk_pure = {'a': params._acoeff, 'b': params._bcoeff}
+            elif isinstance(params, crossFluidParameters):
+                sp1 = spdict[params._species1]
+                sp1.rk_binary[params._species2] = params._acoeff
+                sp2 = spdict[params._species2]
+                sp2.rk_binary[params._species1] = params._acoeff
+
+
+class constantIncompressible(object):
```

Comment:
Why did the class names switch to camelCase here? I guess the previous ones were all lowercase or snake case to match (or nearly match) the names in YAML?

Reply:
All of the class names need to be as they are, since they are the names that are used in existing CTI files. Sorry I missed this one when it was introduced (this is from the recent BinarySolutionTabulatedThermo addition).

Reply:
#658 also addresses this comment.